frontend/src/App.vue

```diff
@@ -56,6 +63,64 @@
             changeDrawerValueTo(value) {
                 this.showDrawer = value;
             },
+
+            keyUpOrDown(event) {
+                let handled = false;
+                this.keymap[event.key] = event.type == "keydown";
+                if (event.ctrlKey && this.keymap["s"]) {
+                    this.saveProject(this.title, true);
+                    handled = true;
+                } else if (event.shiftKey && event.ctrlKey && this.keymap["S"]) {
+                    this.$refs.drawer.$refs.drawerButtons.openSaveProjectModal();
+                    handled = true;
+                } else if (event.ctrlKey && this.keymap["ArrowLeft"]) {
+                    if (this.$refs.mainPage.$refs.debugScene.isRunning) {
+                        this.$refs.mainPage.$refs.debugScene.$refs.navigationPanel.icons[0].action();
+                        handled = true;
+                    }
+                } else if (!event.ctrlKey && this.keymap["ArrowLeft"]) {
```

Comment:
Jeszcze sobie pomyślałem, żeby można było slajdy przesuwać zarówno strzałkami lewo prawo jak i przyciskami pageUp pageDown, więc prośba, żebyś to też tu podpiął.